src/style.css

```diff
@@ -1,3 +1,133 @@
+:root {
+  --link-color: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  height: 60px;
```

Comment:
Be consistent with your vertical margins. It's recommended to add only top or only bottom margin, but not both to prevent potential margin collapse issues. In this case, you might want to remove either 'margin-top' or 'margin-bottom'.